objects-api/src/main/kotlin/com/ritense/objectsapi/taak/TaakObjectConnector.kt

```diff
@@ -128,22 +128,9 @@ class TaakObjectConnector(
         formulierId: String?,
         formulierUrl: String?,
     ): TaakObjectDto {
-        val bsn = bsnProvider?.getBurgerServiceNummer(task)
-        val kvk = kvkProvider?.getKvkNummer(task)
-        val identificaties = mutableListOf<TaakIdentificatie>()
-        if (bsn != null) {
-            identificaties += TaakIdentificatie("bsn", bsn)
-        }
-        if (kvk != null) {
-            identificaties += TaakIdentificatie("kvk", kvk)
-        }
-        if (kvk == null && bsn == null) {
-            throw IllegalStateException("Unable to find the 'Initiator' of task '${task.id}'")
-        }
         val taakObject = TaakObjectDto(
-            bsn = bsn,
-            kvk = kvk,
-            identificatie = identificaties,
+            bsn = bsnProvider?.getBurgerServiceNummer(task),
+            kvk = kvkProvider?.getKvkNummer(task),
```

Comment:
Im not understanding the plan here. This format is no longer supported by the portal. Are we going to delete the TaakConnector?

Reply:
Quoting Thomas: "abandon it"